### DISCUSSION TITLE
[Feature] <Facial Recognition sharing>

### DISCUSSION THREAD

**Discussion by chFresh** [immich-app/immich#8867] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

hoping its possible to implement Facial Recognition sharing so not every member of the family needs to recognize the same people
could be default or closable when partner sharing

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#5089
